admin/release.py

```diff
@@ -852,14 +845,11 @@ def calculate_base_branch(version, path):
     # We create a new branch from a branch, not a tag, because a maintenance
     # or documentation change may have been applied to the branch and not the
     # tag.
-    try:
-        base_branch = [
-            branch for branch in repo.branches if
-            branch.name == base_branch_name][0]
-    except IndexError:
-        raise BaseBranchDoesNotExist()
+    repo.git.checkout(base_branch_name)
```

Comment:
Comment that this is required to make the branch available "locally" for the next step.